### ISSUE TITLE
[feature] Add possibility to ignore barcode/qrcode only

### ISSUE THREAD

**Issue by enricobenedos** [juliuscanute/qr_code_scanner#93]
Good morning,
at the moment this package provide the possibility to scan barcode o qr code with no limitations. 
I think that a very good improvement can be the possibility to choose if we need to scan barcodes or qrcodes indipendently. Something like:
```dart
QRView(
              onlyQr: true,
              //onlyBarCode: true,
              key: _qrKey,
              onQRViewCreated: _onQRViewCreated,
            ),
```
In my opinion this can be a very good feature that can improve a lot the work that every developer need to make in order to distinguish them.

Thank you

**Comment by leoshusar** [comment]
+1. I would like to select like e.g. only QR or only EAN13.

**Comment by bartektartanus** [comment]
Maybe better option is to add `acceptedCodes` which will be enum array of desired code types?
Something like this:
```dart
enum Codes { qr, barcode}

QRView(acceptedCodes: [Codes.qr])
```

**Comment by dspicher** [comment]
PRs are welcome.

**Comment by LeLunZ** [comment]
I made it to controller.setAllowedBarcodeTypes([BarcodeTypes.qr, BarcodeTypes.ean8]).
So you don't need to rebuild the widget if you want to change it.

The possible BarcodeTypes are defined in types/barcode.dart (I took the one which where available in IOS and Android)

**Comment by juliansteenbakker** [comment]
I have released v0.3.0 which adds this function using `formatsAllowed`.

```
    return QRView(
      key: qrKey,
      onQRViewCreated: _onQRViewCreated,
      formatsAllowed: [BarcodeFormat.qrcode],
      overlay: QrScannerOverlayShape(
        borderColor: Colors.red,
        borderRadius: 10,
        borderLength: 30,
        borderWidth: 10,
        cutOutSize: scanArea,
      ),
    );
```
